### PR TITLE
cpp: Document use of 2013-06 schema by bioformats-cpp

### PR DESCRIPTION
**Pull request by rleigh-dundee**
As requested by (I think) @joshmoore

### CODE REVIEW

File: docs/sphinx/developers/cpp/schema.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.. _bf-schema:
+
+OME-XML Schema
+==============
+
+The Bio-Formats C++ implementation currently uses schema version
+:model_doc:`2013-06 </schemas/june-2013.html>` of the OME-XML data
```
Comment (mtbc) [inline docs/sphinx/developers/cpp/schema.txt]: I don't think you need the leading /

File: docs/sphinx/developers/cpp/schema.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+model.  The :doxygen:`model <namespaceome_1_1xml_1_1model.html>` and
+:doxygen:`metadata <namespaceome_1_1xml_1_1meta.html>` interfaces and
+classes are generated from this schema and will read and write OME-XML
+and OME-TIFF files using this version of the schema.  See the
+:ref:`bf-cpp-tutorial` section for further details of these
+interfaces.
+
+The library will be updated to use a newer version of the OME-XML
```
Comment (mtbc) [inline docs/sphinx/developers/cpp/schema.txt]: maybe s/library/implementation/ given that's what we call it above

**Comment by mtbc** [comment]
Staged at http://www.openmicroscopy.org/site/support/bio-formats5.1-staging/developers/cpp/schema.html.

**Comment by mtbc** [comment]
Good to merge once travis is finally placated.

**Comment by joshmoore** [comment]
:green_apple:

**Pull request merged**